code/controllers/configuration.dm

```diff
@@ -236,6 +236,9 @@ var/list/gamemode_cache = list()
 	var/max_acts_per_interval = 140 //Number of actions per interval permitted for spam protection.
 	var/act_interval = 0.1 SECONDS //Interval for spam prevention.
 
+	var/panic_bunker = FALSE //is the panic bunker enabled?
+	var/panic_bunker_message = "Sorry! The panic bunker is enabled. Please head to our discord or forum to get yourself added to the panic bunker bypass."
```

Comment:
Why is this a var on the config controller, but cannot actually be set in the config?

Reply:
oversight, will make it configruable later.

Reply:
`Discord` needs a capital.